assets/js/components/user-input/UserInputPreviewGroup.js

```diff
@@ -193,20 +189,18 @@ export default function UserInputPreviewGroup( {
 				</div>
 			) }
 
-			{ isEditing && (
+			{ isOpen && (
 				<Fragment>
 					<UserInputSelectOptions
 						slug={ slug }
 						max={ USER_INPUT_MAX_ANSWERS[ slug ] }
 						options={ options }
 						alignLeftOptions
 					/>
-					{ errorMessage && (
-						<p className="googlesitekit-error-text">
-							{ errorMessage }
-						</p>
+					{ error && (
```

Comment:
This error is updated on value change (more like on every re-render, but close enough!)